lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -681,7 +681,7 @@ def api_request(method, endpoint, parameters = nil, options = {})
       end
 
       def commit(method, url, parameters = nil, options = {})
-        add_expand_parameters(parameters, options) if parameters
+        add_expand_parameters(parameters, options, method, url) if parameters
```

Comment:
`method` and `url` were added but never used

Reply:
I does get used [here](https://github.com/activemerchant/active_merchant/pull/5360/files#diff-19492101b546175aa3ca6a16ec7d0a610d0adae03cbaf130eff54092dec8e0fbR327) which is why we need to pass them